app/src/main/AndroidManifest.xml

```diff
@@ -36,6 +38,19 @@
             android:name=".screen.ordermanagement.OrderManagementActivity"
             android:screenOrientation="portrait"
             android:theme="@style/AppTheme"/>
+
+        <service android:name=".data.source.local.MyFirebaseMessagingService">
```

Comment:
WARNING: [ExportedService] Exported service does not require permission
 Exported services (services which either set `exported=true` or contain an intent-filter and do not specify `exported=false`) should define a permission that an entity must have in order to launch the service or bind to it. Without this, any application can use this service.